app/Http/Controllers/ReportGroupController.php

```
@@ -65,37 +75,64 @@ public function show(string $id)
+            ], 500);
+        }
     }
 
     /**
      * Remove the specified resource from storage.
      */
     public function destroy(string $id)
     {
-        $reportGroup = ReportGroup::find($id)->whereHas('accounts')->first();
-        if ($reportGroup) {
+        $reportGroup = ReportGroup::find($id);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\ReportGroup' in method 'destroy'. (undefined)

(StaticAccess)